doc/source/parameters/dem/post-processing.rst

```diff
@@ -13,16 +32,20 @@ Lethe-DEM has built-in post-processing capabilities. All post-processing results
 .. important::
  The particles' average velocity and the granular temperature are cell based results calculated instantaneously at each outputting time-step. This means that they are only instantaneous spatial averages, i.e, DO NOT correspond to temporal averages measured between the outputs. For a higher resolution of the results in time, we recommend the use of a lower ``output frequency`` (:doc:`../dem/simulation_control`).
 
-The post-processing subsection of the ``.prm`` file is according to the following example:
+----------------------------
+Force chains post-processing
+----------------------------
+The ``force chains`` feature enables VTU, PTVU, and PVD files to visualize force chains between particles. The name of the generated files are set by the ``output name`` parameter in the simulation_control section plus the suffix ``-force_chains``. The files include:
 
-.. code-block:: text
+* Lines to represent forces between particles in contact. (use wireframe display with paraview)
 
- subsection post-processing
-  # Enable output of grid, granular temperature, and particles' average velocity
-  set Lagrangian post-processing = false
- end
+* The force value for each line.
 
-.. note::
- By default, the post-processing is set to ``false``.
+.. warning::
+ The force chains feature is not implemented for periodic domains. 
+
+The following video shows force chains for the `Rectangular Hopper example <https://chaos-polymtl.github.io/lethe/documentation/examples/dem/rectangular-hopper/rectangular-hopper.html>`_.
+
+ .. raw:: html
 
-* The ``Lagrangian post-processing`` enables the built-in post-processing. The current outputs include the grid of the domain, the granular temperature, and the average velocity of particles (in x, y, z and its norm).
+    <iframe width="600" height="303" src="https://youtu.be/XrXXCz00Yjk?si=6T9_PhBDX3SkUsne" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
Am I the only one that sees that? The link is working well, but this is the only embedded YT video that shows me this. Maybe it is because it's not on the website (master) yet?
![image](https://github.com/chaos-polymtl/lethe/assets/71884806/1c37a1c1-bbee-496a-a80e-15edfc6d0565)

Reply:
Same here. The link is working but I need to open it on another page.

Reply:
I think it's because you did not put the embed youtube video link
It's not the same as directly linking to the youtube video, you need to grab the embedded URL. You can check on youtube directly how to get this URL, but it should have a different prefix.